src/composition/supergraph/config/error/subgraph.rs

```diff
@@ -14,7 +13,7 @@ pub enum ResolveSubgraphError {
         /// The subgraph name that failed to be resolved
         subgraph_name: String,
         /// Supplied path to the supergraph config file
-        supergraph_config_path: Utf8PathBuf,
+        supergraph_config_path: PathBuf,
```

Comment:
__NIT__: why are we changing to `PathBuf` instead of `Utf8PathBuf` here? Wouldn't that risk introducing UTF-8 bugs?